modules/nf-core/ncbidatasets/download/main.nf

```diff
@@ -0,0 +1,108 @@
+process NCBIDATASETS_DOWNLOAD {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "conda-forge::ncbi-datasets-cli=15.11.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ncbi-datasets-pylib:15.11.0--pyhdfd78af_0':
+        'staphb/ncbi-datasets:15.11.0' }"
+
+    input:
+
+    val(meta)
+
+    output:
+    // Required output
+    tuple val(meta), path("*.zip")   ,             emit: zip
+    path "versions.yml"              ,             emit: versions
+    // Optional outputs, depends on tool usage
+    tuple val(meta), path("cds.fna.gz"),           emit: cds,        optional: true
+    tuple val(meta), path("gene.fna.gz"),          emit: gene,       optional: true
+    tuple val(meta), path("*genomic.fna.gz"),      emit: genome,     optional: true
```

Comment:
Some software throw an error with `fna` extension, maybe consider renaming to `f[ast]a` to be safe.
Example from MerquryFK: 
```
Cannot open GCA_903798025.1.fna as a .cram|[bs]am|f{ast}[aq][.gz]|db|dam file
```

Reply:
I see! I am wondering though if I should change the filenames as obtained from `datasets`. Another way in dealing with this problem is renaming the file during the nextflow process, e.g. using `stageAs`

Reply:
Generally I put `stageAs` in the module itself, so that would be changing a bunch of existing modules.

Reply:
That makes sense. I'll ask for feedback on this issue in Slack

Reply:
IMO I would leave as fna. The software taking the fna file can stage it with whatever extension is necessary